blocks/api/parser.js

```diff
@@ -165,8 +170,13 @@ export function getBlockAttributes( blockType, rawContent, attributes ) {
  * @return {?Object}            An initialized block object (if possible)
  */
 export function createBlockWithFallback( name, rawContent, attributes ) {
-	// Use type from block content, otherwise find unknown handler.
-	name = name || getUnknownTypeHandlerName();
+	// If unknown type, use unknown handler and apply paragraph formatting.
+	if ( undefined === name ) {
+		rawContent = formatting.autop( rawContent );
```

Comment:
Hm, don't we already do that here: https://github.com/WordPress/gutenberg/blob/v1.1.0/blocks/library/freeform/old-editor.js#L79?

Reply:
>Hm, don't we already do that here: 

Good call, I did not see this, and it would likely be redundant. An issue with leaving the current implementation is that it needlessly flags the post as dirty because the original value of the block content does not have `autop` applied, only after setup has completed (in master, focusing and blurring a freeform block without changes will mark the post as needing save).

With ideas floating around server-side autop, I'll consider whether the existing `autop` call in `OldEditor` is still necessary.